src/pages/ProfilePage.tsx

```diff
@@ -139,7 +141,7 @@ function ProfilePage({route}: ProfilePageProps) {
 
     return (
         <ScreenWrapper testID={ProfilePage.displayName}>
-            <FullPageNotFoundView shouldShow={CONST.RESTRICTED_ACCOUNT_IDS.includes(accountID)}>
+            <FullPageNotFoundView shouldShow={shouldShowBlockingView || CONST.RESTRICTED_ACCOUNT_IDS.includes(accountID)}>
```

Comment:
+ the other suggestion
```suggestion
            <FullPageNotFoundView shouldShow={shouldShowBlockingView}>
```